pom.xml

```diff
@@ -209,6 +209,9 @@
                         <exclude>CVE-2022-24823</exclude>
                         <exclude>CVE-2022-2047</exclude>
                         <exclude>CVE-2022-30187</exclude>
+                        <!-- org.yaml.snakeyaml (1.33) https://ossindex.sonatype.org/vulnerability/CVE-2022-38750
+                        Currently not fixed in latest version, however a test dependency and not used on untrusted files.
+                        Can be removed once org.yaml.snakeyaml upgrades to a newer version. -->
```

Comment:
Only as (optional) comment: False positive in snakeyaml. According to https://bitbucket.org/snakeyaml/snakeyaml/issues/531/stackoverflow-oss-fuzz-47081 this is already fixed in 1.32.